clang/utils/TableGen/ClangAttrEmitter.cpp

```diff
@@ -3694,6 +3694,11 @@ static void GenerateMutualExclusionsChecks(const Record &Attr,
     }
   }
 
+  // If there are any decl or stmt attributes, silence -Woverloaded-virtual
+  // warnings for them both.
+  if (!DeclAttrs.empty() || !StmtAttrs.empty())
```

Comment:
NB for other reviewers: this was a cherry pick from community and silences a bunch of warning diagnostics that were making it hard to read the CI pipeline output.

Reply:
Cherry-pick from :  llvm/llvm-project@65c22ac#diff-58111c7c545672c3a67b1b31a37d4103b35d9c584a46c74b0be14cc37741491b